apps/src/blockly/components/ModalFunctionEditor.tsx

```diff
@@ -7,14 +7,15 @@ import {
 import moduleStyles from './modal-function-editor.module.scss';
 import classNames from 'classnames';
 import Button from '@cdo/apps/templates/Button';
-import msg from '@cdo/locale';
+import {commonI18n} from '@cdo/apps/types/locale';
```

Comment:
Can you confirm all `tsx` files use `commonI18n` instead of `msg`?

Reply:
Yep! As of #53623 this is an exported object with the specific name `commonI18n`.